lambdas/access_counts/index.py

```diff
@@ -322,6 +325,11 @@ def handler(event, context):
         # We start from scratch, so make sure we don't have any old data.
         delete_dir(QUERY_RESULT_BUCKET, OBJECT_ACCESS_LOG_DIR)
 
+    # We can't write more than 100 days worth of data at a time due to Athena's partitioning limitations.
+    # Moreover, we don't want the lambda to time out, so just process 100 days and let the next invocation handle the rest.
+    if (end_ts - start_ts).days >= MAX_OPEN_PARTITIONS:
```

Comment:
why `>=` and not `>`? i would probably do the same thing, just asking :)

Reply:
Hm, actually, neither of them is right...

Suppose `start_ts` is day=1, hour=23, and `end_ts` is day=3, hour=1. The difference is 1 day (plus two hours), while the time range spans 3 days.

Reply:
Good point. Don't you guarantee elsewhere that end_ts is on a date boundary so that the code never writes incomplete days? In that case I think the worst case is MAX_OPEN_PARTITIONS - 2 days right?